package.json

```diff
@@ -27,7 +27,8 @@
     "clean": "gatsby clean",
     "test:links": "remark src/pages --ignore-pattern src/pages/graphql,src/pages/_includes --quiet --frail",
     "build:spectaql": "spectaql --target-dir static/graphql-api/ --target-file index.html spectaql/config.yml",
-    "dev:spectaql": "spectaql --development-mode-live spectaql/config.yml"
+    "dev:spectaql": "spectaql --development-mode-live spectaql/config.yml",
+    "lint": "docker run --rm -e RUN_LOCAL=true --env-file '.github/super-linter.env' -v \"$PWD\":/tmp/lint github/super-linter:slim-v4.10.1"
```

Comment:
I get the following error when I run `yarn lint`:

```
2023-03-08 18:15:07 [INFO]   --------------------------------------------
2023-03-08 18:15:07 [INFO]   Gathering GitHub information...
2023-03-08 18:15:07 [INFO]   NOTE: ENV VAR [RUN_LOCAL] has been set to:[true]
2023-03-08 18:15:07 [INFO]   bypassing GitHub Actions variables...
2023-03-08 18:15:07 [INFO]   Linting all files in mapped directory:[/tmp/lint]
2023-03-08 18:15:07 [INFO]   Successfully found:[GITHUB_TOKEN]
2023-03-08 18:15:07 [INFO]   --------------------------------------------
2023-03-08 18:15:07 [INFO]   Gathering user validation information...
2023-03-08 18:15:08 [FATAL]     -> MARKDOWN_LINTER_RULES rules file (/action/lib/.automation/.markdownlint.yml) doesn't exist. Terminating...
```

Is it expected?

Reply:
It is possible, because when you run it locally it runs on the entire codebase.
The issue with adding linters separately is that if we add the markdownlinter, then we should also add yamllint and gitleaks and potentially all other linters we will enable in the SuperLinter. And to make them work as in SuperLinter we need to install the versions that correspond to the superlinter and configure each linter correspondingly. Eventually, it will make no sense to use SuperLinter if we configure all linters separately anyway. Supporting separate linters in package.json will make maintenance of every project much more expensive.
I'd rather keep it to writers on how to configure separate linters locally: IDE extensions, CLI tools, Docker, etc.

Reply:
Ah, `[FATAL]     -> MARKDOWN_LINTER_RULES rules file (/action/lib/.automation/.markdownlint.yml) doesn't exist.` this means it didn't find the configuration file `.markdownlint.yml`. I'll try to reproduce.

Reply:
I couldn't reproduce:
```
2023-03-08 18:39:03 [INFO]   Gathering GitHub information...
2023-03-08 18:39:03 [INFO]   NOTE: ENV VAR [RUN_LOCAL] has been set to:[true]
2023-03-08 18:39:03 [INFO]   bypassing GitHub Actions variables...
2023-03-08 18:39:03 [INFO]   Linting all files in mapped directory:[/tmp/lint]
2023-03-08 18:39:03 [INFO]   Successfully found:[GITHUB_TOKEN]
2023-03-08 18:39:03 [INFO]   --------------------------------------------
2023-03-08 18:39:03 [INFO]   Gathering user validation information...
2023-03-08 18:39:18 [INFO]   ----------------------------------------------
2023-03-08 18:39:18 [INFO]   User provided file:[/tmp/lint/.github/linters/.markdownlint.yml] exists, setting rules file...
2023-03-08 18:39:29 [INFO]   ----------------------------------------------
2023-03-08 18:39:29 [INFO]   User provided file:[/tmp/lint/.github/linters/.yaml-lint.yml] exists, setting rules file...
```

Reply:
🤷